### PR TITLE
Added WithHostNetwork func to the pod pkg

### PR DESCRIPTION
**Pull request by evgenLevin**

### CODE REVIEW

File: pkg/pod/pod.go [hunk 1]
```diff
@@ -705,6 +705,25 @@ func (builder *Builder) WithSecondaryNetwork(network []*multus.NetworkSelectionE
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to pod's definition.
+func (builder *Builder) WithHostNetwork() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Defining pod's HostNetwork")
```
Comment (kononovn) [inline pkg/pod/pod.go:714]: Please change logs to 
glog.V(100).Infof("Applying HostNetwork flag to pod's %s configuration", builder.Definition.Name)

**Review by kononovn — CHANGES_REQUESTED** [review]
Just 1 small comment.

**Comment by evgenLevin** [comment]
@kononovn PTAL

**Review by kononovn — APPROVED** [review]
lgtm

**Pull request merged**